### PR TITLE
fix(scully): fix Could not connect to server

### PR DESCRIPTION
**Pull request by jorgeucano**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
Fix problem with Could not connect to server

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #40 

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by SanderElias** [comment]
I added a few things to this pr. @jorgeucano you might want to review this one too.

File: scully.config.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ require('./extraPlugin/voidPlugin');
 exports.config = {
   /** projectRoot is mandatory! */
   projectRoot: './projects/sampleBlog/src/app',
-  /** outFolder is where the static distribution files end up */
-  outFolder: './dist/static',
```
Comment (d-koppenhagen) [inline scully.config.js]: as it is renamed: Probably a `ng-update` migration script is needed?

Comment (SanderElias) [reply scully.config.js]: Good catch. 
No, we are in Aplha, an ngUpdate script isn't needed, but the correct default is.

File: scully/renderPlugins/puppeteerRenderPlugin.ts [hunk 2]
```diff
@@ -63,3 +69,12 @@ export const puppeteerRender = async (route: HandledRoute): Promise<string> => {
 function waitForIt(milliSeconds) {
   return new Promise(resolve => setTimeout(() => resolve(), milliSeconds));
 }
+
+const windowSet = (page, name, value) =>
```
Comment (aaronfrost) [inline scully/renderPlugins/puppeteerRenderPlugin.ts]: This adds to the scullyVersion to the HTML? Or what element gets the attribute?

File: scully/startBackgroundServer.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  const binary = ['/dist/scully/scully', '/node_modules/.bin/scully', '/node_modules/@scullyio/scully/scully']
+    .map(p => join(scullyConfig.homeFolder, p + '.js'))
+    .find(p => existsSync(p));
+
+  if (!binary) {
+    logError('Could not find scully binaries');
+    process.exit(15);
+    return;
+  }
+  console.log('starting', binary);
```
Comment (aaronfrost) [inline scully/startBackgroundServer.ts]: Should this console.log be `log` instead? Or removed?

File: scully/renderPlugins/puppeteerRenderPlugin.ts (hunk 2 above)
Comment (SanderElias) [reply scully/renderPlugins/puppeteerRenderPlugin.ts]: it adds it to the body tag, as an attribute.

File: scully/startBackgroundServer.ts (hunk 3 above)
Comment (SanderElias) [reply scully/startBackgroundServer.ts]: I think this is a left-over, I think it should be removed.

**Pull request merged**